taskiq/compat.py

```diff
@@ -13,8 +14,12 @@
 if IS_PYDANTIC2:
     T = TypeVar("T")
 
+    @lru_cache(maxsize=None)
+    def create_type_adapter(annot: T) -> pydantic.TypeAdapter:
```

Comment:
Should be

```suggestion
    def create_type_adapter(annot: T) -> pydantic.TypeAdapter[T]:
```

Reply:
Oh yeah, I forgot it. Thanks for pointing out it.